tequilapi/client/client.go

```diff
@@ -106,16 +106,28 @@ func (client *Client) Disconnect() (err error) {
 	return nil
 }
 
+// ConnectionStatistics returns statistics about current connection
+func (client *Client) ConnectionStatistics() (statistics StatisticsDTO, err error) {
+	response, err := client.http.Get("connection/statistics", url.Values{})
+	if err != nil {
+		return
+	}
+	defer response.Body.Close()
+
+	err = parseResponseJson(response, &statistics)
+	return
+}
+
 // Status returns connection status
-func (client *Client) Status() (status StatusDto, err error) {
+func (client *Client) Status() (status StatusDTO, err error) {
 	response, err := client.http.Get("connection", url.Values{})
 	if err != nil {
 		return
 	}
 	defer response.Body.Close()
 
 	err = parseResponseJson(response, &status)
-	return status, err
+	return
```

Comment:
Is it bad to have clearer declaration?

Reply:
Since result parameters are named, I don't see much value added by rewriting the same named parameters here which is the default of `return`

Reply:
Removed named parameters to reduce discussions on this.